source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -2243,6 +2248,10 @@ export abstract class Action {
     parse(json: any) {
         this.id = json["id"];
         this.title = json["title"];
+        this.isPrimaryAction = false;
```

Comment:
Not necessary

Reply:
This is outdated code. It is not from latest commit